CMakeLists.txt

```diff
@@ -24,9 +24,6 @@ include(FeatureSummary)
 include(CompilerSettings)
 include(OMPLUtils)
 
-set(CMAKE_RUNTIME_OUTPUT_DIRECTORY "${PROJECT_BINARY_DIR}/bin")
-set(CMAKE_LIBRARY_OUTPUT_DIRECTORY "${PROJECT_BINARY_DIR}/lib")
-
```

Comment:
This might break some existing expectations, but it also makes it easier for the `setup.py` script to find the resulting binaries.

https://github.com/ompl/ompl/pull/1103/files#diff-60f7d754a0c159e4789a1ac359dc6552c6187ec903af6e6c3af43232be7449a8R102-R104